packages/docs/src/components/landing/Partners.vue

```diff
@@ -0,0 +1,163 @@
+                <path d="M73.2484 18.2728H63.8542V21.5779H69.2084C69.0624 23.2551 68.4297 24.5376 67.3588 25.5735C66.288 26.5601 64.9251 27.0534 63.1728 27.0534C62.1993 27.0534 61.2744 26.8561 60.4956 26.5108C59.6682 26.1655 58.9867 25.6722 58.4026 24.9816C57.8185 24.3403 57.3805 23.551 57.0397 22.6138C56.699 21.6765 56.553 20.6899 56.553 19.5553C56.553 18.4208 56.699 17.4342 57.0397 16.4969C57.3318 15.5597 57.8185 14.8197 58.4026 14.1291C58.9867 13.4878 59.6682 12.9945 60.4956 12.5999C61.3231 12.2546 62.1993 12.0573 63.2215 12.0573C65.2658 12.0573 66.8234 12.5506 67.8942 13.5371L70.4253 10.972C68.527 9.49217 66.0933 8.70288 63.2215 8.70288C61.6152 8.70288 60.1549 8.94955 58.8407 9.49217C57.5265 10.0348 56.407 10.7254 55.4821 11.6626C54.5573 12.5999 53.8272 13.7345 53.3405 15.0664C52.8537 16.3983 52.6104 17.8781 52.6104 19.4567C52.6104 21.0352 52.8537 22.5151 53.3891 23.847C53.9246 25.1789 54.606 26.3135 55.5308 27.2507C56.4557 28.188 57.5752 28.9279 58.8894 29.4212C60.2036 29.9638 61.6639 30.2105 63.2215 30.2105C64.779 30.2105 66.1906 29.9638 67.4562 29.4212C68.7217 28.8786 69.7925 28.188 70.6687 27.2507C71.5448 26.3135 72.2263 25.1789 72.713 23.847C73.1998 22.5151 73.4432 21.0352 73.4432 19.4567V18.8154C73.2971 18.7167 73.2484 18.4701 73.2484 18.2728Z" fill="black" />
+              </g>
+              <defs>
+                <clipPath id="clip0_2161_3062">
+                  <rect width="159.538" height="30.4274" fill="white" />
+                </clipPath>
+              </defs>
+            </svg>
+          </a>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  name: 'LandingPartners',
+}
+</script>
+
+<style lang="scss" scoped>
+@import "src/assets/main";
+
+.partners {
+  width: 100%;
+  position: relative;
+  padding: 3rem 0;
+  border-top: 1px solid #f2f2f2;
+  border-bottom: 1px solid #f2f2f2;
+  background: #f4f9fc8c;
+
+  &__wrapper {
+    @include wrapper();
+  }
+
+  &__inner {
+    @include row-flex();
+
+    align-items: center;
+  }
+
+  &__title {
+    text-align: center;
+    width: 100%;
+    font-size: 1.4rem;
+  }
+
+  &__logos {
+    @include size(12);
+
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+
+    .logo {
+      margin-top: 1.3rem;
+      word-wrap: break-word;
+      margin-left: 15px;
+      margin-right: 15px;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: flex-start;
+    }
+  }
+}
+
+.item {
```

Comment:
This one is also a bit weird (see https://github.com/epicmaxco/vuestic-ui/pull/2218/files#r943522232)